DependencyInjection/Configuration.php

```diff
@@ -486,6 +488,10 @@ public function getConfigTreeBuilder()
                             ->scalarNode('title')->defaultNull()->end() // pushover
                             ->scalarNode('host')->defaultNull()->end() // syslogudp & hipchat
                             ->scalarNode('port')->defaultValue(514)->end() // syslogudp
+                            ->enumNode('rfc')
+                                ->values([SyslogUdpHandler::RFC5424, SyslogUdpHandler::RFC3164])
```

Comment:
using raw value (cf snippet bellow) would allow people to use the RFC, without triggering exception for composer 1

```suggestion
                                ->values([SyslogUdpHandler::RFC5424, SyslogUdpHandler::RFC3164, 2 /* SyslogUdpHandler::RFC5424e */])
```

Reply:
https://github.com/symfony/monolog-bundle/pull/343#discussion_r470972897 here @Tobion suggest to not to use plain values, which point of view should I adhere to

Reply:
It's always better to use constant, that's why @Tobion suggested to not use plain value when a constant exists.

In this case (and only for the value `2`), we can not use constant, as the constant does not exist in monolog 1.
But we should not prevent people that use monolog 2 to use this value.
As a result, you should either remove this ENUM constraint and let people enter whatever they want, or allows this rfc by using a plain value.

Reply:
Ok, i'll change it to scalarNode with default value